app/helpers/application_helper.rb

```diff
@@ -16,4 +16,8 @@ def preview_url(slug, auth_bypass_id: nil)
 
     url
   end
+
+  def live_url(slug)
+    "#{Plek.new.external_url_for('www')}/#{slug.sub(/^\//, '')}"
```

Comment:
Plek has a helper that does this. You should only need to use `Plek.new.website_root`.

Where is `live_url` being used?

Reply:
Would website root give you the `www` domain? For example `/learn-to-drive-a-car` => `https://www.gov.uk/learn-to-drive-a-car`

Reply:
Yep! On integration:

```
irb(main):001:0> Plek.new.website_root
=> "https://www.integration.publishing.service.gov.uk"
```